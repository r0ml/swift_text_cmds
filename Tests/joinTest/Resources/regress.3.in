1901,Jean Henri Dunant,M
1901,Frederic Passy,M
1902,Elie Ducommun,M
1905,Baroness Bertha Sophie Felicita Von Suttner,F
1910,Permanent International Peace Bureau,
