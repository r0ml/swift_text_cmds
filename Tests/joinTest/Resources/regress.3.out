Jean Henri Dunant,1901,M,Switzerland
Frederic Passy,1901,M,France
Elie Ducommun,1902,M,Switzerland
Baroness Bertha Sophie Felicita Von Suttner,1905,F
